Server/service/pingService.js

```diff
@@ -70,16 +70,13 @@ const startPingService = async (db) => {
 const cleanup = async () => {
   try {
     logger.info("Cleaning up Ping Service...", { service: SERVICE_NAME });
+
     // Clean out the queue
     const jobs = await queue.getJobs(["active", "waiting", "delayed"]);
-    for (const job of jobs) {
-      await job.remove();
-    }
+    await Promise.all(jobs.map((job) => job.remove()));
 
     // Close workers
-    for (const worker of workers) {
-      await worker.close();
-    }
+    await Promise.all(workers.map((worker) => worker.close()));
 
     // Close the queue
     await queue.close();
```

Comment:
Good for freeing up memory and CPU usage. Good use of async.

Reply:
Ya my biggest concern is running out of resources since we could have a very large number of monitors we need to ping conccurently

Reply:
> Ya my biggest concern is running out of resources since we could have a very large number of monitors we need to ping conccurently

I would say we don't have to think deeply about those two: 

- Multi-tenancy
- Scalability

We can always get those sorted out once there is an MVP we can play with.

Reply:
Agreed, we shouldn't get too bogged down in performance details at this point.  Plenty of time for that once things are up and running :laughing: